src/PointwiseFunctions/AnalyticSolutions/NewtonianEulerSolutions/IsentropicVortex.hpp

```diff
@@ -91,14 +100,21 @@ class IsentropicVortex {
     static constexpr OptionString help = {"The mean flow velocity."};
   };
 
+  /// The amplitude of the perturbation generating a source term.
+  struct SourceStrength {
```

Comment:
I don't understand this name. Do you mean `epsilon`? If so, I think `PerturbationAmplitude` or `PerturbationStrength` would be better. Or maybe the documentation is confusing because I read the documentation as saying that `W` is the source, while `\epsilon sin(z)` is a perturbation to the source. I guess what you should do depends on whether it's the documentation that needs clarification or the variable names.

If it's `SourceStrength` -> `PerturbationAmplitude` please rename the variables throughout.

Reply:
Good point. `epsilon` is technically the amplitude of the perturbation, and, since the resulting source is proportional to the derivative of the perturbation, `epsilon` is also the amplitude or strength of the resulting source. 

I just did `SourceStrength` -> `PerturbationAmplitude` and added a comment in the documentation. Hope it's clearer now.